### ISSUE TITLE
Cannot sort top-level notebooks via drag & drop

### ISSUE THREAD

**Issue by CoeJoder** [laurent22/joplin#10474]
### Operating system

Linux

### Joplin version

2.14.22

### Desktop version info

Joplin 2.14.22 (prod, linux)

Client ID: c0236ef1e39b43bfb54c20930ecb12f6
Sync Version: 3
Profile Version: 46
Keychain Supported: No

Revision: e579eb956

Backup: 1.4.0
Reset Checkboxes: 1.0.1
Rich Markdown: 0.13.4
Search & Replace: 2.2.0

### Current behaviour

1. Left click and hold on a notebook
2. Drag down
3. Release: it always becomes a subnotebook rather than changing its ordering a the top-level

### Expected behaviour

Expected ability to drag-and-drop to another position at top-level.

**Comment by personalizedrefrigerator** [comment]
I've added the `enhancement` label as this doesn't seem to be supported. Currently, the only notebook sorting options are "title" and "updated date":
![screenshot: View menu, sort notebooks by submenu](https://github.com/laurent22/joplin/assets/46334387/9f10a097-ec2b-4b1d-838c-4aac109e21f6)

**Comment by KushyHuK** [comment]
First I want to say thank you for your work !

In order to completely replace other note app It should have a feature to custom sort notebooks via drag & drop as notes already do.

This feature has been requested in [#206](https://github.com/laurent22/joplin/issues/206) and [#461](https://github.com/laurent22/joplin/issues/461) since 2018, 7 years now I'm done waiting for this..

What do you mean by for now @laurent22 have you any clue when it could be implemented? [your post from 2020](https://discourse.joplinapp.org/t/no-custom-order-for-sort-notebooks-by/10209/8)

Here is what we're expecting:

<img width="420" height="462" alt="Image" src="https://github.com/user-attachments/assets/2c04c151-29b6-489d-a69b-2e61e3e14c08" />
<img width="410" height="473" alt="Image" src="https://github.com/user-attachments/assets/96d84adc-7918-42c1-9479-272cca0fcd41" />

I can add that "number the notebooks and sort by title" is really really messy with over 200 notes

**Comment by PiewieP** [comment]
I agree that having a 'custom order sort' option for notes but not for nodebooks is not something that I expected. Being able to custom sort nodebooks by dragging / dropping just like you do with notes would come in handy for sure!

**Comment by sbakker** [comment]
+1 for this feature.

Been hunting for this feature myself. All related issues for this have been closed over the years or marked as duplicates of an issue that resulted in the implementation of the "title" and "updated date" sort options, but _not_ the "custom order".

**Comment by zhizhongpu** [comment]
same issue on Mac os

Info:

Copyright © 2016-2026 Laurent Cozic
Joplin 3.5.13 (prod, darwin)

Device: darwin, Apple M2 Pro
Client ID: bce356290ea540d0bd4d8c12ac44b8d2
Sync Version: 3
Profile Version: 49
Keychain Supported: Yes
Alternative instance ID: -

Revision: 0c1511f

Backup: 1.5.1
Inline TODO: 2.0.5
Menu items, Shortcuts, Toolbar icons: 1.1.0
